### PR TITLE
docs: update the metrics as a table

### PR DESCRIPTION
**Pull request by SataQiu**
Signed-off-by: SataQiu <qiushida@beyondcent.com>

### Ⅰ. Describe what this PR did
docs: update the metrics as a table

### CODE REVIEW

**Comment by Starnop** [comment]
Could you please add a column in the table and  stuff metrics type into it?

**Comment by SataQiu** [comment]
> 
> 
> Could you please add a column in the table and stuff metrics type into it?

Sure! Done.

File: docs/user_guide/metrics.md [hunk 1]
```diff
@@ -4,34 +4,40 @@ This doc contains all the metrics that Dragonfly components currently support. N
...
-- dragonfly_supernode_schedule_duration_milliseconds{peer} - duration for task scheduling in milliseconds.
-- dragonfly_supernode_cdn_trigger_total{} - total times of triggering cdn. counter type.
-- dragonfly_supernode_cdn_trigger_failed_total{} - total failed times of triggering cdn. counter type.
-- dragonfly_supernode_cdn_cache_hit_total{} - total times of hitting cdn cache. counter type.
-- dragonfly_supernode_cdn_download_total{} - total times of cdn downloading. counter type.
-- dragonfly_supernode_cdn_download_failed_total{} - total failure times of cdn downloading. counter type.
-- dragonfly_supernode_pieces_downloaded_size_bytes_total{} - total size of pieces downloaded from supernode in bytes. counter type.
-- dragonfly_supernode_gc_peers_total{} - total number of peers that have been garbage collected. counter type.
-- dragonfly_supernode_gc_tasks_total{} - total number of tasks that have been garbage collected. counter type.
+Name                                                                     | Type      | Description
```
Comment (Starnop) [inline docs/user_guide/metrics.md]: So embarrassed. 😢 And could you please add a column in the table and stuff labels into it?

File: docs/user_guide/metrics.md [hunk 2]
```diff
@@ -4,34 +4,40 @@ This doc contains all the metrics that Dragonfly components currently support. N
...
+:----------------------------------------------------------------------- | :-------- | :----------
+dragonfly_supernode_build_info{version, revision, goversion, arch, os}   | gauge     | build and version information of supernode.
```
Comment (yeya24) [inline docs/user_guide/metrics.md]: I hope we can separate the metric name and its labels. So please add another column called labels

File: docs/user_guide/metrics.md [hunk 3]
```diff
@@ -4,34 +4,40 @@ This doc contains all the metrics that Dragonfly components currently support. N
...
+dragonfly_supernode_http_requests_total{code, handler, method}           | counter   | total number of http requests.
```
Comment (yeya24) [inline docs/user_guide/metrics.md]: Please capitalize all these metrics description sentences.

Comment (SataQiu) [reply docs/user_guide/metrics.md]: Done

**Comment by Starnop** [comment]
LGTM.

Good job. THX.

**Pull request merged**